fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/SelectItemColumn.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019
+ * (C) Copyright IBM Corp. 2019,2021
```

Comment:
probably doesn't need a copyright year

Reply:
pls explain

Reply:
sorry I meant... needs 2019... since it's a new year...